invenio_records_resources/version.py

```diff
@@ -13,4 +13,4 @@
 and parsed by ``setup.py``.
 """
 
-__version__ = "0.8.8"
+__version__ = "0.9.0"
```

Comment:
comment: talk maybe to @zzacharo or @kprzerwa because they are doing some work on this module as part of the sprint, just to make sure it does not collide.

Reply:
@fenekku from my side I thin is fine but let's talk once you are online again! Btw, invenio-app-rdm has already released in `0.9.0`.